pom.xml

```diff
@@ -2170,6 +2170,22 @@
                     </excludes>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>de.qaware.maven</groupId>
+                <artifactId>go-offline-maven-plugin</artifactId>
```

Comment:
Can this be in `pluginManagement` or in some profile, if it's only used in the CI?